fenix/app/src/main/res/values/strings.xml

```diff
@@ -529,6 +529,11 @@
     <string name="preferences_open_links_in_apps_never">Never</string>
     <!-- Preference for open download with an external download manager app -->
     <string name="preferences_external_download_manager">External download manager</string>
+    <!-- Preference for enabling gecko engine logs -->
+    <string name="preferences_enable_gecko_logs">Enable Gecko logs</string>
+    <!-- Message to indicate users that we are quitting the application to apply the changes -->
+    <string name="quit_application">Quitting the application to apply changes…</string>
```

Comment:
Instead of automatically quitting to apply changes I would make the toast say "Changes will apply on the next restart" with maybe an action button that says "Quit now".

But since this is a secret setting I'm not concerned if you want to 🚢  I'm good with this too

Reply:
I just reused the same text that we had for other similar options 😅, I'll land for now latter we could improve the text in a follow up :)  

https://github.com/mozilla-mobile/firefox-android/blob/a2c278068c525d68c7daa90bc306fb8525728fdf/fenix/app/src/main/res/values/strings.xml#L553
https://github.com/mozilla-mobile/firefox-android/blob/a2c278068c525d68c7daa90bc306fb8525728fdf/fenix/app/src/main/res/values/strings.xml#L842
https://github.com/mozilla-mobile/firefox-android/blob/a2c278068c525d68c7daa90bc306fb8525728fdf/fenix/app/src/main/res/values/strings.xml#L475C94-L475C102